doc/en/components/grids/_shared/export-excel.md

```diff
@@ -163,23 +165,10 @@ function exportEventFreezeHeaders(grid: IgrGridBaseDirective, args: IgrExporterE
   </IgrGridToolbarActions>
 </IgrGridToolbar>
 ```
-
-<!-- React -->
-<!-- ComponentStart: HierarchicalGrid -->
-```tsx
-function exportEventFreezeHeaders(sender: IgrGridToolbarExporter, args: IgrExporterEventEventArgs) {
-    args.detail.options.freezeHeaders = true;
-}
-
-<IgrGridToolbar>
-  <IgrGridToolbarActions>
-    <IgrGridToolbarExporter exportStarted={exportEventFreezeHeaders}></IgrGridToolbarExporter>
-  </IgrGridToolbarActions>
-</IgrGridToolbar>
-```
-<!-- ComponentEnd: HierarchicalGrid -->
+<!-- ComponentEnd: Grid, TreeGrid, HierarchicalGrid -->
 <!-- end: React -->
 
+<!-- ComponentStart: Grid -->
```

Comment:
You can add `<!-- ComponentStart: Grid, TreeGrid -->...<!-- ComponentEnd: Grid, TreeGrid -->` and remove this section [here](https://github.com/IgniteUI/igniteui-xplat-docs/pull/1259/files#diff-ea1a6774d128994b4b3cc022bdbca4748db8bda2ad56ee4648b70bdc18074a13R209-R225) since the text above the snippet explains how the `freezeHeaders` option works.